templates/qcbm-optimization.yaml

```diff
@@ -40,6 +40,8 @@ spec:
             from zquantum.core.bitstring_distribution import load_bitstring_distribution
 
             distance_measure = get_func_from_specs({{inputs.parameters.distance-measure-specs}})
+            distance_measure_parameters = eval("{{inputs.parameters.distance-measure-parameters}}")
```

Comment:
I think you don't need `eval`, you can just do `{{inputs.parameters.distance-measure-parameters}}`, without the `"`. See [here](https://github.com/zapatacomputing/z-quantum-optimizers/blob/7e56e27d462b63556d4f581f06dab77755b6b779/templates/optimizers.yaml#L65).

Reply:
Addressed [here](https://github.com/zapatacomputing/z-quantum-qcbm/pull/14/commits/3a4f373b40fcb0362934aead7c2eb5f2fd628a4b)